src/main/java/com/iota/iri/service/milestone/impl/MilestoneSolidifierImpl.java

```diff
@@ -319,23 +315,9 @@ public void addMilestoneCandidate(Hash milestoneHash, int milestoneIndex) {
         if (!unsolidMilestones.containsKey(milestoneHash) && !seenMilestones.containsKey(milestoneIndex) &&
                 milestoneIndex > getLatestSolidMilestoneIndex()) {
             unsolidMilestones.put(milestoneHash, milestoneIndex);
-            updateQueues(milestoneHash, milestoneIndex);
```

Comment:
shouldn't we still call `scanMilestonesInQueue`?

Reply:
That we should. A scan happens after each round of processing, but it's still best to scan on new additions as well.